hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/txns/contract/ContractCallTransitionLogicTest.java

```diff
@@ -222,6 +241,55 @@ void verifyExternaliseContractResultCallEth() {
         inOrder.verify(worldState).getCreatedContractIds();
     }
 
+    @Test
+    void failsOnMissingReceiverSigReq() {
+        givenValidTxnCtx();
+        // and:
+        given(accessor.getTxn()).willReturn(contractCallTxn);
+        // and:
+        senderAccount.initBalance(1234L);
+        given(accountStore.loadAccount(senderAccount.getId())).willReturn(senderAccount);
+        given(accountStore.loadContract(new Id(target.getShardNum(), target.getRealmNum(), target.getContractNum())))
```

Comment:
I have to look this up - in a `given` what does an argument which looks like `new Foo(...)` mean?  It really has to match the exact expression (including explicit calls to get the 3 tuple pieces from the target)?  It's a bit surprising to me you can target a method call via a `given` so specifically ...